root/layout/components/sidebar/SidebarAcousticBrainz.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import hydrate from '../../../utility/hydrate';
+
+import {SidebarProperty, SidebarProperties} from './SidebarProperties';
+
+const SidebarAcousticBrainz = ({recording}: {recording: RecordingT}) => {
+  const [count, setCount] = React.useState(0);
+  const [isCountLoaded, setIsCountLoaded] = React.useState(false);
+  const [data, setData] = React.useState({});
+  const [isDataLoaded, setIsDataLoaded] = React.useState(false);
+
+  function fetchAcousticBrainzCount() {
+    const countUrl = `//acousticbrainz.org/api/v1/${recording.gid}/count`;
+    fetch(countUrl).then(
+      resp => resp.json(),
+    ).then(
+      data => {
+        setCount(data.count);
+        setIsCountLoaded(true);
+        if (data.count) {
+          fetchAcousticBrainzData();
+        }
+      },
+    );
+  }
+
+  function fetchAcousticBrainzData() {
+    const dataUrl = `//acousticbrainz.org/api/v1/${recording.gid}/low-level`;
+    fetch(dataUrl).then(
+      resp => resp.json(),
+    ).then(
+      data => {
+        setData(data);
+        setIsDataLoaded(true);
+      },
+    );
+  }
+
+  React.useEffect(fetchAcousticBrainzCount, [recording.gid]);
+  return (isCountLoaded && count) ? (
+    <>
+      <h2>
+        {l('Acoustic information')}
+      </h2>
+
+      <a className="external" href={`//acousticbrainz.org/${recording.gid}`}>
+        {l(`${count} submissions on AcousticBrainz`)}
```

Comment:
To handle plural forms for numeric values we'd use `ln` instead of `l` here, and interpolate the count this way
```suggestion
        {texp.ln(
          '{count} submission on AcousticBrainz',
          '{count} submissions on AcousticBrainz',
          {count},
        )}
```